sci-libs/shapelib/files/0020-This-patch-fixes-test-running-on-Gentoo.-Perhaps-it-.patch

```diff
@@ -0,0 +1,67 @@
+From 07cb77c93fc9e43ad4a244c192ee7a86b9d163e0 Mon Sep 17 00:00:00 2001
+From: Tanuki-no Torigava <vmikhailikov@gmail.com>
+Date: Fri, 6 Jun 2025 16:09:44 +0300
+Subject: [PATCH] This patch fixes test running on Gentoo. Perhaps it could fix
```

Comment:
The patch should explain what failed and why. It also needs a link to the upstream PR.